ckanext/unaids/helpers.py

```diff
@@ -77,6 +79,10 @@ def get_bulk_file_uploader_default_fields():
     return toolkit.config.get(BULK_FILE_UPLOADER_DEFAULT_FIELDS, {})
 
 
+def get_ape_url():
+    return config.get("ckanext.unaids.ape_url", "") + "?return_url=" + full_current_url()
```

Comment:
What do you think about using urllib for generting this URL with params?

Reply:
Ok, I've used urllib.